content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Using HPE GreenLake Console's API Gateway for Data Services Cloud Console
+date: 2021-07-29T14:28:17.779Z
+featuredBlog: false
+priority: 1
+author: Ron Dharma
+authorimage: /img/dscc-icon.png
+thumbnailimage: ""
+tags:
+  - data-services-cloud-console
+---
+## Secured yet Agile
+
+One major principle of Application Programming Interface (API) for the HPE Data Services Cloud Console (DSCC) is secured, but also flexible for consumption of multiple applications or tools that use this API to extend their function using the DSCC. One of many features for the delegated authorization below includes the ability to provide access to the API resources inside the DSCC (such as volumes, arrays, etc.) without embedding the user's credential (username, and password) in the API communication stream.
```

Comment:
Hi, Ron... Dale, the lay person here, providing  input on the comprehensibility of some things in this post :-). I see that you're adding in some introductory information, and that is wonderful. Thank you. I think some minor changes to wording might help in its readability, so if it's okay with you, I'll make some suggestions.

I think what you're trying to say in this paragraph is:
HPE GreenLake offers an API for those who wish to monitor their HPE GreenLake storage activities via the HPE GreenLake Data Services Cloud Console. It is called the HPE Data Services Cloud Console (DSCC) Application Programming Interface (API). This API aims to securely, but flexibly, extend the function of multiple applications or tools using DSCC. One of the key ways it does so is through the ability to provide access to the API resources inside the DSCC (such as volumes, arrays, etc.) without embedding the user's credentials (username and password) in the API communication stream.

(now, the above was my interpretation of what you wrote. If I didn't get that right, we need to modify it. If I did get it right, I recommend you use my rewording ;-) )